lib/engine/matchers/contains_any_matcher.rb

```diff
@@ -0,0 +1,15 @@
+module SplitIoClient
+  class ContainsAnyMatcher < SetMatcher
+    def self.matcher_type
+      'CONTAINS_ANY'.freeze
+    end
+
+    def initialize(attribute, remote_array)
+      super(attribute, remote_array)
+    end
+
+    def match?(_key, data)
+      local_set(data, @attribute).intersect? @remote_set
```

Comment:
No, here's a test for that: https://github.com/splitio/ruby-client/blob/feature/set_matchers/spec/engine/matchers/contains_any_matcher_spec.rb#L13